package.json

```diff
@@ -28,7 +27,7 @@
     "prebuild:js": "cp node_modules/plotly.js/dist/plotly.min.js dash_core_components_base/plotly.min.js",
     "build:js": "webpack --mode production",
     "build:py_and_r": "dash-generate-components ./src/components dash_core_components -p package-info.json && cp dash_core_components_base/** dash_core_components/ && dash-generate-components ./src/components dash_core_components -p package-info.json --r-prefix 'dcc'",
-    "build": "run-s build:js generator build:py_and_r",
```

Comment:
Some extra clean up / simplification, this is no longer necessary b/c of the `prebuild` command.